Sources/MapboxNavigation/NavigationMapView.swift

```diff
@@ -134,6 +134,20 @@ open class NavigationMapView: UIView {
         }
     }
     
+    /**
+     Toggles displaying alternative routes when in active navigation session.
+     
+     If enabled, view will draw actual alternative route lines on the map.
+     Default value is `true`.
+     */
+    public var showsContinuousAlternatives: Bool = true {
```

Comment:
Is this property essentially a feature flag, so that we can disable it in CarPlay, or are there other reasons for wanting to enable or disable continuous alternative route display?

Reply:
Reworked it to be a "feature flag" on NavigationViewController